test/core/ticktree/TickTreeMarketOrder.t.sol

```diff
@@ -14,8 +14,10 @@ import "mgv_lib/Debug.sol";
 //   - a middle tick
 //   - a higher tick
 // 2. we take a snapshot of Mangrove's tick tree
-// 3. we run a market order in Mangrove and remove the corresponding offers in the snapshot tick tree
-// 4. we check that Mangrove's tick tree matches the test tick tree.
+// 4. we remove the offers the market order should take from the snapshot tick tree
```

Comment:
4.?

Reply:
doh. fixed in a33e9f33a0d1f83269620ba7e531f6ea6bd63bd6